nilearn/plotting/tests/test_find_cuts.py

```diff
@@ -32,6 +32,14 @@ def test_find_cut_coords():
                                # pass. x, y, z = [24.75, 3.17, 9.875]
                                rtol=6e-2)
 
+    # regression test (cf. #473)
+    # some values in image used to crash largest_connected_component()
+    rng = np.random.RandomState(42)
+    data = (rng.randn(36, 43, 36) * 10**-7)
```

Comment:
minor comment: `* 1e-7`

More importantly isn't this test supposed to fail without your fix in find_cuts.py? Spoiler: it doesn't seem to, at least for me ...